gadgets/advise_seccomp/gadget.yaml

```diff
@@ -12,3 +12,5 @@ datasources:
     annotations:
       cli.supported-output-modes: advise
       cli.default-output-mode: advise
+paramDefaults:
+  operator.oci.ebpf.map-fetch-interval: "0"
```

Comment:
BTW, I'm not a big fan of using fully qualified names here as `params` section uses a diff notation. Idk, it looks strange that I have to specify `operator.oci` on one and not in the other, don't you think?

```yaml
params:
  ebpf:
    ppid:
      key: ppid
      description: Parent process ID
paramDefaults:
  operator.oci.ebpf.ppid: "999"
```

Reply:
Yeah, me neither. 

As a more general thought I believe we should get rid of them `params`, I don't see its utility. The description can be added in the `datasources.<dsname>.fields.<fieldname>.description`, the default value can be change in the eBPF code and the key .... mmmm why would I need to change the key? 😅 ... Anyway, that not related with this PR.

Reply:
Let's see what other folks think about it